site/content/contribute/getting-started/code-review.md

```diff
@@ -38,47 +38,65 @@ If you are a core committer seeking a review
 
 1. Submit your pull request.
     * Follow the [contribution checklist](https://developers.mattermost.com/contribute/getting-started/contribution-checklist/).
-2. Assign a [product manager](/contribute/getting-started/core-committers/#product-managers) to your review and label your pull request with `1: PM Review`.
-    * If your changes do not affect the user experience, you may skip this step.
-    * Product managers ensure the changes meet [user experience guidelines](https://docs.mattermost.com/developer/fx-guidelines.html).
-    * Wait for their review before continuing so as to avoid churn if changes are requested.
-    * Note that product managers may assign core committers after completing their own review.
-    * The PM should remove `1: PM Review` when their review is done.
-3. Assign two [core committers](/contribute/getting-started/core-committers/) to your review and label your pull request with `2: Dev Review`.
-    * When picking your first core committer, consider someone with domain expertise relative to your changes. Sometimes GitHub will recommend a recent editor of the code, but often you must rely on your own intuition from past interactions.
-    * When picking your second core committer, consider someone from another team who may have good or expert knowledge of the language and/or problem you're solving, even if they aren't intricately familiar with the codebase. This can provide a fresh set of eyes on the code to reveal blindspots that are not biased by hitting deadlines, and helps expose the team to new parts of the code to help spread out domain knowledge. This may not make sense for every pull request but is a practice to keep in mind.
-    * Don't be afraid to pick someone who gives "hard" reviews. Code review feedback is never a personal attack: it should "sharpen" the skills of both the author and the reviewers, not to mention improving the quality of the product.
-    * If you are Mattermost staff, try to take into account the timeoff calendar.
-    * Try to avoid assigning the same person to all of your reviews unless they are related.
-    * When in doubt, ask for recommendations on our community server.
-4. Assign a [QA tester](/contribute/getting-started/core-committers/#qa-testers) and label your pull request with `2: QA Review`.
-    * It is the PM or QA tester's responsibility to determine the scope of required testing, if any.
-    * Reviews by QA may occur at the same time as review by core committers. Be sure to ask for a second review as needed when changes are made.
-    * The QA tester should remove the `2: QA Review` label and add the `QA Review Done` label when their review is done, or if a review is deemed not necessary.
-5. Apply additional labels as necessary:
+2. Immediately assign `1: PM Review`, `2: Dev Review`, and `3: QA Review`.
```

Comment:
Perhaps "Immediately add labels" rather than "Immediately assign", in order to make explicitly distinct from assigning reviewers?